<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>telaFiltrar</class>
 <widget class="QDialog" name="telaFiltrar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filtrar</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>343</width>
     <height>127</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="check_wifi">
        <property name="text">
         <string>Wi-Fi</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_ar">
        <property name="text">
         <string>Ar condicionado</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_tv">
        <property name="text">
         <string>TV</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_sacada">
        <property name="text">
         <string>Sacada</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="check_roupao">
        <property name="text">
         <string>Roupão e chinelos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_higiene">
        <property name="text">
         <string>Produtos de higiene</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="check_cofre">
        <property name="text">
         <string>Cofre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_banheira">
        <property name="text">
         <string>Banheira</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_cafe">
        <property name="text">
         <string>Café da manhã</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="check_servico">
        <property name="text">
         <string>Serviço de quarto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_miniBar">
        <property name="text">
         <string>Mini bar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>telaFiltrar</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>telaFiltrar</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
